local-libs/traceviewer-libs/react-components/src/components/generic-xy-output-component.tsx

```diff
@@ -68,12 +69,18 @@ interface GenericXYState extends AbstractTreeOutputState {
     allMax: number;
     allMin: number;
     cursor: string;
+    timelineUnit: string;
+    timelineUnitType: TimelineUnitType;
 }
 
+type TimelineUnitType = 'time' | 'cycles' | 'bytes' | 'calls' | 'bytes/sec' | 'iterations/sec';
+
 interface GenericXYProps extends AbstractOutputProps {
     formatX?: (x: number | bigint | string) => string;
     formatY?: (y: number) => string;
     stacked?: boolean;
+    timelineUnit?: string;
```

Comment:
Those are never set in any of the existing code (nor the above). Nor the other ones of the props.